### ISSUE TITLE
可以用中国的智谱AI  或者讯飞的 API key 么

### ISSUE THREAD

**Issue by 2m897** [saoudrizwan/claude-dev#321]

**Comment by specter119** [comment]
一些个人观点：
这个插件对模型能力要求很高，可以尝试先用用ollama里的开源模型试试，比如 `MFDoom/deepseek-coder-v2-tool-calling`, `qwen2` 等，会发现体验很不好。
ollama 里也有智谱的 glm4: <https://ollama.com/library/glm4>，但是我没尝试。
我有尝试 gpt-4o，勉强能用。
换句话说，现在这个插件强依赖 claude sonnet，剩下的模型都不建议。

---

Some personal opinions:
This extension has very high requirements for model capabilities. You can try to use the open source models in ollama first, such as `MFDoom/deepseek-coder-v2-tool-calling`, `qwen2`, etc., and you will find that the experience is not very good.
Ollama also has Zhipu's glm4: <https://ollama.com/library/glm4>, but I didn't try it.
I tried gpt-4o, which barely worked.
In other words, this extesion now strongly relies on Claude Sonnet, and the rest of the models are not recommended.

**Comment by saoudrizwan** [comment]
If these APIs are OpenAI Compatible, you can use them in Settings > OpenAI compatible. Otherwise I'll need some direction about how these APIs work/their docs and if they have SDKs I can use so I can consider adding them as API providers. Closing for now, but feel free to comment here with additional information.
